README.md

```diff
@@ -1,40 +1,21 @@
-Instagram Challenge
-===================
+# Instragram Challenge
 
-## Instructions
+*[Exercise instructions](instructions.md)*
 
-* Challenge time: one weekend
-* Feel free to use Google, your notes, books, etc., but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+## Installation instructions
 
-## Task
+Clone this repo. Then:
 
-Build Instagram: Simple huh!
-
-Your challenge is to build Instagram using Rails. You'll need **users** who can post **pictures**, write **comments** on pictures and **like** a picture. Style it like Instagram's website (or more awesome).
-
-Bonus if you can add filters!
-
-## How to start
-
-1. Produce some stories, break them down into tasks, and estimate
-2. Fork this repo, clone, etc
-3. Initialize a new rails project
-
-Remember to proceed in small steps! Getting confused? Make the steps even smaller.
+```
+bundle install
+bin/rails db:create
+bin/rails db:migrate
 
-## Code Quality
+bundle exec rspec # Run the tests to ensure it works
```

Comment:
Tests failing cos of old UsersController

Reply:
fixed